index.html

```diff
@@ -211,7 +211,7 @@
         <h1>Glossary</h1>
         <dl>
             <dt><dfn>active buttons state</dfn></dt>
-                <dd>The condition when a pointer has a non-zero value for the <code>buttons</code> property. For mouse, this is when the device has at least one button depressed. For touch, this is when there is physical contact with the digitizer. For pen, this is when the pen has physical contact with the digitizer.</dd> 
+                <dd>The condition when a pointer has a non-zero value for the <code>buttons</code> property. For mouse, this is when the device has at least one button depressed. For touch, this is when there is physical contact with the digitizer. For pen, this is when the pen either has physical contact with the digitizer, or has at least one button depressed while hovering.</dd>
```

Comment:
"...or at least one button *is* depressed while hovering"

It may also need an extra qualifier at the end of that sentence, along the lines of "(for hardware that supports this)"? Or do all digitizers with pens that have buttons intrinsically able to recognize these buttons? (the answer here may be yes, though perhaps even then the qualifier may be useful to have)